lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -621,6 +618,11 @@ namespace Js
         }
 #endif
 
+// xplat-todo: This __try/__except catches AV and resumes from out of bound TypedArray access.
```

Comment:
Thanks for referring to the related virtualAlloc dependencies. From the code, IsValidVirtualBufferLength always returns false and JavascriptArrayBuffer::AllocWrapper already has Assert(false) on non _WIN64 platforms. This feature is already disabled. I'll change the comment and remove this xplat-todo.